index.html

```diff
@@ -0,0 +1,136 @@
+              <a class="nav-link" href="#Hello_world">Hello world</a>
+              <hr />
+            </li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main id="main-doc">
+        <section id="Introduction" class="main-section">
+          <h2>Introduction</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="what_you_should_already_know" class="main-section">
+          <h2>what you should already know</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="JavaScript_and_Java" class="main-section">
+          <h2>JavaScript and Java</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+        </section>
+        <section id="Hello_world" class="main-section">
+          <h2>Hello world</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <ul>
+            <li>Selam</li>
+            <li>Hello</li>
+            <li>Guten Morgon</li>
+            <li>Merhaba</li>
+            <li>Kaifa</li>
+          </ul>
+        </section>
+        <section id="Variables" class="main-section">
+          <h2>Variables</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+            <code></code>
+            <code></code>
+            <code></code>
+            <code></code>
+            <code></code>
```

Comment:
Please note unused HTML tags here. You can either remove them or fill them with intended content.